### ISSUE TITLE
GPU scaling better blending needed?

### ISSUE THREAD

**Issue by Ink230** [Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java#475]
At higher resolution scaling, "tiling" becomes apparent from the assets used to build the battle stage. Note, on 1x you can see some horizontal lines that are most likely due to bad asset design (but need to confirm retail).

Should better blending be implemented at 1x, 2x, 4x?
- Possible?
- What type of blending?
- What will make it look consistent regardless of the scale setting chosen?

Floor at 4x
![image](https://user-images.githubusercontent.com/29797557/236644212-e6217583-6a1c-4f50-8c79-7eabd1c96104.png)

Floor at 1x
![image](https://user-images.githubusercontent.com/29797557/236644221-34fb91ff-541e-4262-b47b-a1609a96d627.png)

**Comment by LordMonoxide** [comment]
What do you mean by "blending"?

**Comment by Ink230** [comment]
Whatever makes the repeated tiling of the ground texture *blend* together without the gridlines appearing.

The ground texture *looks* like it was made to tile perfectly but some errors in positioning or rendering cause them to not tile perfectly. Except maybe one portion of the texture that will never be seamless.

![image](https://github.com/Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java/assets/29797557/22655fef-21af-453c-8de3-8ce89a4f3738)

Emulator on the right.

You can see emulation has better blending of the horizontal gridlines of the texture tiling. The vertical lines are better in some places but worse in others in comparison (debatable).

The notable horizonal portion that looks subpar is
![image](https://github.com/Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java/assets/29797557/c13e8083-1f36-4df6-b0ca-dfb9ada70aae)

compared to

![image](https://github.com/Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java/assets/29797557/2d7b7e7d-76df-4758-93e3-837b065edac7)

In summary
• Something isn't quite right with the tiling
• Perhaps we can also tile things better for no gridlines

By blending, and not knowing what I'm talking about
• Is the positioning of the tiling accurate / the best it can be?
• Is there any tile ground texture pre-processing / post-processing effects to crudely or elegantly *blur*, *dither*, or *line up* the edges of textures better?

**Comment by LordMonoxide** [comment]
I don't think these minor quality issues are worth spending time on since this is only temporary until opengl, and opengl will just do a much better job of rendering than we could ever do in software
